core/server/web/shared/middlewares/image/handle-image-sizes.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const sharp = require('sharp');
```

Comment:
We have a wrapped image manipulator library (`lib/image/manipulator`). _It wraps sharp and e.g. has a functionality to check if sharp is physically installed. IMO we should use this lib.

I have not reviewed this PR, because a lot of people are already involved here. Just noticed this tiny thing while scrolling over :)

Reply:
Good spot, this is important as sharp is an optional dependency.

Reply:
Nice catch - due to this working with raw buffers rather than files on disk - i'm not sure if it belongs in `lib/image` but maybe we make another manipulator in there - either way it needs to protect against sharp missing :+1:

Reply:
Another point for going with another wrapper or reusing/adjusting `lib/image/manipulator` would be not getting the same compression [issues](https://github.com/TryGhost/Ghost/issues/10144) we were having with pre-compressed images

Reply:
oh tasty - thanks :+1: